ev3dev2/motor.py

```diff
@@ -2237,30 +2224,103 @@ def follow_gyro_angle(self,
 
         self.stop()
 
-    def turn_to_angle_gyro(self,
+    def _turn(
+            self,
             speed,
-            target_angle=0,
+            target_angle,
+            brake=True,
             wiggle_room=2,
             sleep_time=0.01
         ):
         """
-        Pivot Turn
+        Use a GyroSensor to rotate in place for ``target_angle``
 
         ``speed`` is the desired speed of the midpoint of the robot
 
-        ``target_angle`` is the target angle we want to pivot to
+        ``target_angle`` is the number of degrees we want to rotate
+
+        ``brake`` hit the brakes once we reach ``target_angle``
 
         ``wiggle_room`` is the +/- angle threshold to control how accurate the turn should be
 
         ``sleep_time`` is how many seconds we sleep on each pass through
             the loop.  This is to give the robot a chance to react
             to the new motor settings. This should be something small such
             as 0.01 (10ms).
+        """
+
+        # MoveTank does not have information on wheel size and distance (that is
+        # MoveDifferential) so we must use a GyroSensor to control how far we rotate.
+        assert self._gyro, "GyroSensor must be defined"
```

Comment:
Will fix here but am also going to file a bug for us to clean this up elsewhere...we have a lot of places we are doing asserts where it would be more clear if we were raising ValueError.

Reply:
Filed #707